scripts/release-tag.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+function current_version {
+    local str
+    for v in MAJOR MINOR PATCH; do
+        str+=$(sed -rn "s/set\(OPENSMT_VERSION_${v} ([^ )]*)\)/\1/p" CMakeLists.txt).
```

Comment:
`sed -r` doesn't work on Mac's standard `sed`.  It could be good to avoid using the extension.

Reply:
Also, does this write to `CMakeLists.txt`?  Isn't that what `-r` does?

Reply:
`-r` stands for extended regular expressions. I will remove it ... The script does not overwrite `CMakeLists.txt`, but I will make it more clear.